src/Example.tsx

```diff
@@ -2,11 +2,19 @@ import { FC, useState } from 'react';
 import { Button } from '@chakra-ui/react';
 import styled from '@emotion/styled';
 import TimeSelectBottomSheet from 'domain/moit/TimeSelectBottomSheet';
+import useGetCheckIsFirst from 'domain/study/hooks/useGetCheckIsFirst';
 import { useModal } from 'hooks/useModal';
+
+const DUMMY_SUTDY_ID = 1;
```

Comment:
```suggestion
const DUMMY_STUDY_ID = 1;
```
오타긴 한데 더미라 안고쳐도 될듯 ㅎㅎ